{{ cookiecutter.project_slug }}/.env.example

```diff
@@ -0,0 +1,7 @@
+{% if cookiecutter.data_dir == 'local' %}
+DATA_DIR=data/
+{% elif cookiecutter.data_dir == 'github' %}
+DATA_DIR=data/
+{% elif cookiecutter.data_dir == 'box' %}
+DATA_DIR=~/Library/CloudStorage/Box-Box/
```

Comment:
I might add a comment or other examples (not sure what's standard for an .env.example) for Windows, WSL. 

If you're on windows its "C:\Users\<USERNAME>\Box\"
If you're on WSL its `/mnt/c/Users/<USERNAME>/Box`

Probably not super important unless students are using box

Reply:
Good point. If we could do branching prompts in the cookiecutter, I would have a follow-up to ask operating system, but I believe that's not supported? Since that's not a thing, I'm happy to just leave it as-is, with the expectation that folks will edit it to the correct path.